.github/workflows/ci.yaml

```diff
@@ -406,3 +406,60 @@ jobs:
         with:
           name: release-images
           path: ./release-images
+
+  trivy-scan-image:
+    runs-on: ubuntu-20.04
+    needs: docker-amd64
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Download release images
+        uses: actions/download-artifact@v2
+        with:
+          name: release-images
+          path: ./release-images
+
+      - name: Run Trivy vulnerability scanner in image mode
+        uses: aquasecurity/trivy-action@master
```

Comment:
suggest using the version tag here, even though we're always running with the latest version of trivy, we can at least avoid breakages in the Action itself by pinning

Reply:
Oh right! I always forget. Nice catch!